packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -39,10 +39,18 @@
         background-color: var(--general-main-1);
         button {
             height: 4rem;
-            @include mobile-or-tablet-screen {
```

Comment:
@hirad-deriv this change is not correct, we need to keep the mobile and tablet exactly the same and max the content to 600px.

Reply:
I made changes based on the planning of TH tablet view.